The hpy seems to be not needed to run tests.

--- dist-meta-0.8.1/tests/requirements.txt.orig
+++ dist-meta-0.8.1/tests/requirements.txt
@@ -8,7 +8,6 @@
 coverage>=5.1
 first>=2.0.2
 flake8==3.9.2
-hpy==0.0.3; python_version >= "3.7" and implementation_name == "cpython" and python_version < "3.11"
 importlib-metadata>=3.6.0
 pypi-json>=0.2.1
 pytest>=6.0.0
